package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "db:init": "prisma migrate reset",
     "db:seed": "prisma db seed --preview-feature",
-    "db:up": "docker-compose --file docker/docker-compose.yml up --detach postgres",
+    "db:up": "sudo docker-compose --file docker/docker-compose.yml up --detach postgres",
```

Comment:
its only sudo for your machine I think, on macOS and windows it should not be sudo. So I don't think the change is good